docs/_includes/custom-head.html

```diff
@@ -4,3 +4,5 @@
   1. Head over to https://realfavicongenerator.net/ to add your own favicons.
   2. Customize default _includes/custom-head.html in your source directory and insert the given code snippet.
 {% endcomment %}
+
+CUSTOM
```

Comment:
Is this related to #2355?

Reply:
No - a change that will happen but snuck into this PR instead of the other one.